### DISCUSSION TITLE
Trial License

### DISCUSSION THREAD

**Discussion by bloesss** [rustdesk/rustdesk-server#305] [General]
We would like to test the "Pro" Features, is there any trial License?

**Comment by dinger1986** [comment]
please contact sales@rustdesk.com

**Reply by bloesss** [reply]
I just have, thank you !

**Comment by paspo** [comment]
Also, some docs/screenshots of the pro features would be a great idea!

PS: the "documentation" links on the [pricing page](https://rustdesk.com/pricing.html) (free plan and personal plan) aren't working.

**Reply by rustdesk** [reply]
Thanks a lot. Fixed.

**Reply by rustdesk** [reply]
@paspo do you want a license? I can send a permanent license to you, since you have helped us a lot.
